docs/es/week09/09-1.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
This change wasn't strictly necessary for this PR, but I added it so that the numbers matched in the local preview.

Reply:
This stuff is going to create merge conflicts. Please, remove these boilerplates.

Reply:
Done